### ISSUE TITLE
Feature: Add a character countdown and warning to the alt text field

### ISSUE THREAD

**Issue by alundgard** [projectblacklight/spotlight#3486]
### Problem being solved
As currently implemented, [widgets with the alt text field](https://github.com/projectblacklight/spotlight/issues/3164) do not give any indication of a recommended character limit. Accessibility guidelines tend to recommend 100-150 characters.

### Possible design approach
Show a character countdown in or below the alt text field, and then show a warning beyond the 150 character count: "Alt text should not exceed 150 characters.”

<img width="1113" height="368" alt="Image" src="https://github.com/user-attachments/assets/aad6ed88-6a72-45a6-ab67-84cd0206e571" />

<img width="1113" height="368" alt="Image" src="https://github.com/user-attachments/assets/fb6c8025-2477-4fe8-ad52-7ef3ef0266dc" />
